src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/AddAzureVpnClientRevokedCertificateCommand.cs

```diff
@@ -59,10 +59,6 @@ public override void Execute()
         {
 
             base.Execute();
-            if (!this.IsVirtualNetworkGatewayPresent(ResourceGroupName, VirtualNetworkGatewayName))
```

Comment:
@tylerd Hey Tyler, rather than removing the exception, why not use

```cs
throw new ArgumentException(string.Format(Microsoft.Azure.Commands.Network.Properties.Resources.ResourceNotFound, VirtualNetworkGatewayName));
```

That way the name of the resource is actually displayed rather than `{0}`.